pywinauto/mouse.py

```diff
@@ -238,8 +239,24 @@ def right_click(coords=(0, 0)):
     _perform_click_input(button='right', coords=coords)
 
 
-def move(coords=(0, 0)):
+def move(coords=(0, 0), duration=0):
```

Comment:
I'd make it `0.0` by default and `raise TypeError("duration must be float (in seconds)")` if it's not float. So the test with `assertRaises` is also necessary.

Reply:
added, 9338419d5c4f3bfe5619e7b899069ff2fe4c13bb